earth_enterprise/src/common/SharedString.h

```diff
@@ -79,6 +79,10 @@ class SharedString {
       return strStore.size();
     }
 
+    static uint64 GetSharedStringSize() {
+      return sizeof(key) + sizeof(strStore);
```

Comment:
I would take out `sizeof(strStore)` - since it's static there's only one of them, so it doesn't really affect the size of individual strings.